packages/components/menu/src/menu.vue

```diff
@@ -97,6 +77,8 @@ export default defineComponent({
     const alteredCollapse = ref(false)
     const rootMenuEmitter = mitt()
     const router = instance.appContext.config.globalProperties.$router
+    const menu = ref(null)
+    const filteredSlot = ref(slots.default?.())
```

Comment:
Can we avoiding invoking slot expression that's not in `render` function?

Reply:
The initial value can be an empty array
```suggestion
    const filteredSlot = ref([])

```